server/feeders/reviewsgenerator.js

```diff
@@ -139,22 +127,22 @@ var triggerRestaurantsRatings = function() {
     console.log('Total restaurants modified: ' + restaurantsModified);
   };
 
-  utils.getListByType('Review', null, fiwareHeaders)
-  .then(function(data) {
-    reviews = data.body;
-    Object.keys(restaurantsData).forEach(function(element, index) {
-      var restaurantName = restaurantsData[index].name;
-      restaurantReviews = utils.getRestaurantReviews(
-        restaurantName,
-        reviews);
+  Object.keys(restaurantsData).forEach(function(element, index) {
+    var restaurantName = restaurantsData[index].name;
+    var sqlList = [];
```

Comment:
can we name this `filter`, instead of sqlList? variable name is strange as it is not SQL actually